keentools/geotracker/pinmode.py

```diff
@@ -236,17 +239,20 @@ def _init_pinmode(self, area: Area, context: Optional[Any]=None) -> None:
         GTLoader.update_viewport_shaders(area, normals=settings.lit_wireframe)
 
         geotracker = get_current_geotracker_item()
+        self.camera_clip_start = geotracker.camobj.data.clip_start
         self.camera_clip_end = geotracker.camobj.data.clip_end
-        if GTConfig.auto_increase_far_clip_distance:
-            changed, dist = change_far_clip_plane(
-                geotracker.camobj, geotracker.geomobj,
-                prev_clip_end=self.camera_clip_end)
-            if changed:
-                default_txt = deepcopy(vp.texter().get_default_text())
-                default_txt[0]['text'] = f'Camera far clip plane ' \
-                                         f'has been changed to {dist:.1f}'
-                default_txt[0]['color'] = (1.0, 0.0, 1.0, 0.85)
-                GTLoader.viewport().message_to_screen(default_txt)
+        if GTConfig.auto_increase_far_clip_distance and geotracker.camobj and \
+                change_near_and_far_clip_planes(geotracker.camobj, geotracker.geomobj,
+                                                prev_clip_start=self.camera_clip_start,
+                                                prev_clip_end=self.camera_clip_end):
+            near = geotracker.camobj.data.clip_start
+            far = geotracker.camobj.data.clip_end
+            default_txt = deepcopy(vp.texter().get_default_text())
+            default_txt[0]['text'] = f'Camera clip planes ' \
```

Comment:
Don't copy paste text

Reply:
It's dictionary that stores the text and its formating:
Here is type hint:
get_default_text(self) -> List[Dict]